osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineBreak.cs

```diff
@@ -204,12 +204,12 @@ private void updateState()
             {
                 bool active = IsHovered || IsDragged;
 
-                var colour = colours.GreyCarmineLighter;
```

Comment:
Uh... sure about this? This colour *was* matching bottom timeline:

https://github.com/ppy/osu/blob/1933c14bee5dfd9ef02656d229dcd8ffcecddd11/osu.Game/Screens/Edit/Components/Timelines/Summary/Parts/BreakPart.cs#L23-L32

Purple is kiai.

https://github.com/ppy/osu/blob/1933c14bee5dfd9ef02656d229dcd8ffcecddd11/osu.Game/Screens/Edit/Components/Timelines/Summary/Parts/EffectPointVisualisation.cs#L101